clang/test/dpct/cuda-math-intrinsics.cu

```diff
@@ -1,4 +1,4 @@
-// RUN: dpct --format-range=none -out-root %T/cuda-math-intrinsics %s --cuda-include-path="%cuda-path/include" -- -x cuda --cuda-host-only --std=c++14
+// RUN: dpct --format-range=none --use-dpcpp-extensions=intel_device_math -out-root %T/cuda-math-intrinsics %s --cuda-include-path="%cuda-path/include" -- -x cuda --cuda-host-only --std=c++14
```

Comment:
How about keeping the default migration lit case?

Reply:
fixed